Touch Bar Simulator/util.swift

```diff
@@ -179,6 +179,13 @@ extension TargetActionSender {
 			self.action = #selector(ActionTrampoline<Self>.performAction)
 		}
 	}
+	func addAction(_ action: @escaping ((Self) -> Void)) {
+		let lastAction = self.onAction
+		self.onAction = { sender in
+			lastAction?(sender)
+			action(sender)
+		}
+	}
```

Comment:
The problem with doing it like this is that there's no way to add ability to remove an action. I think a better solution would be to store an array of action handlers using associated object.